service/src/main/java/com/codeforcommunity/processor/ProcessorImpl.java

```diff
@@ -39,20 +31,31 @@ public ProcessorImpl(DSLContext db) {
     this.db = db;
   }
 
+  // News
+
   @Override
   public List<NewsReturn> getAllNews() {
     List<NewsReturn> news = db.selectFrom(Tables.NEWS).fetchInto(NewsReturn.class);
     return news;
   }
 
   @Override
-  public Optional<NewsReturn> createNews(String title, String description, String imageUrl, String author, LocalDateTime date, String content) {
+  public Optional<NewsReturn> createNews(String title, String description, String imageUrl, String author,
+      LocalDateTime date, String content) {
     try {
-      Result result = db.insertInto(Tables.NEWS, Tables.NEWS.TITLE, Tables.NEWS.DESCRIPTION, Tables.NEWS.IMAGE_URL, Tables.NEWS.AUTHOR, Tables.NEWS.DATE,
-              Tables.NEWS.CONTENT).values(title, description, imageUrl, author, Timestamp.valueOf(date), content)
-              .returning(Tables.NEWS.ID)
-              .fetch();
-      return getNews((int) result.getValue(0,0));
+
+      NewsRecord newsToCreate = db.newRecord(Tables.NEWS);
+
+      newsToCreate.setTitle(title);
+      newsToCreate.setDescription(description);
+      newsToCreate.setImageUrl(imageUrl);
+      newsToCreate.setAuthor(author);
+      newsToCreate.setDate(Timestamp.valueOf(date));
```

Comment:
This is kind of tangential to this PR but just so you know in your database schema you can make it so that dates default to the time they are inserted. Idk if that's what you want here but if it is lmk and I'll show you how to set that up